compat-tests.el

```diff
@@ -3007,6 +3007,23 @@
       (should (eq (compat-call copy-tree x) x))
       (should (eq (compat-call copy-tree x t) x))))
 
+  ;; TODO reenable tests when Emacs snapshot is updated to include new `copy-tree'
```

Comment:
Wouldn't it be better to pull these tests out and use `skip-unless` to do version checks?  That would make logs more readable.

Reply:
This was only a temporary measure. The check has already been removed.